### PR TITLE
feat(TDI-40637): read one document

### PR DESCRIPTION
**Pull request by Voievidko**
https://jira.talendforge.org/browse/TDI-40637

### CODE REVIEW

File: couchbase/src/test/java/org/talend/components/couchbase/CouchbaseUtilTest.java [hunk 1]
```diff
@@ -12,154 +12,55 @@
...
-        private double col8 = Double.MIN_VALUE;
-
-        private double col9 = Double.MAX_VALUE;
-
-        private boolean col10 = Boolean.TRUE;
-
-        private ZonedDateTime col11 = ZonedDateTime.of(2018, 10, 30, 10, 30, 59, 0, ZoneId.of("UTC"));
-
-        private List<String> col12 = new ArrayList<>(Arrays.asList("data1", "data2", "data3"));
+        // Bucket bucket = COUCHBASE_CONTAINER.getCouchbaseCluster().openBucket(BUCKET_NAME, BUCKET_PASSWORD);
```
Comment (ypiel-talend) [inline couchbase/src/test/java/org/talend/components/couchbase/CouchbaseUtilTest.java]: Remove commented code

File: couchbase/src/main/java/org/talend/components/couchbase/source/CouchbaseInputConfiguration.java [hunk 2]
```diff
@@ -28,7 +29,9 @@
 
 @Version(1)
```
Comment (ypiel-talend) [inline couchbase/src/main/java/org/talend/components/couchbase/source/CouchbaseInputConfiguration.java]: Is it still version 1 ?

Comment (Voievidko) [reply couchbase/src/main/java/org/talend/components/couchbase/source/CouchbaseInputConfiguration.java]: yes. Seems we not update version when change component for now.

Comment (rmannibucau) [reply couchbase/src/main/java/org/talend/components/couchbase/source/CouchbaseInputConfiguration.java]: side note: you "broke" (selectAction)  the configuration so you must increment the version + implement a migration handler

Comment (Voievidko) [reply couchbase/src/main/java/org/talend/components/couchbase/source/CouchbaseInputConfiguration.java]: @rmannibucau thanks for comment. Will do it

File: couchbase/src/test/java/org/talend/components/couchbase/CouchbaseUtilTest.java (hunk 1 above)
Comment (Voievidko) [reply couchbase/src/test/java/org/talend/components/couchbase/CouchbaseUtilTest.java]: Done

File: couchbase/pom.xml [hunk 3]
```diff
@@ -35,5 +35,11 @@
       <version>${testcontainers.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```
Comment (Voievidko) [inline couchbase/pom.xml]: todo: remove hamcrest dependency

**Pull request closed without merging**